src/features/projectsV2/ProjectsSection.tsx

```diff
@@ -31,6 +32,8 @@ const ProjectsSection = ({ isMobile }: ProjectsSectionProps) => {
   } = useProjects();
   const { mapOptions, updateMapOption } = useProjectsMap();
   const [tabSelected, setTabSelected] = useState<ProjectTabs>('topProjects');
+  const { tenantConfig } = useTenant();
+  const isSalesforceTenant = tenantConfig.config.slug === 'salesforce';
```

Comment:
Please generalize this logic. We will not be showing the tabs when `topProjectsOnly = true`. The salesforce tenant is an example of this, but there can be others.